spec/models/form_profile_spec.rb

```diff
@@ -1858,6 +1859,24 @@ def expect_prefilled(form_id)
                 end
               end
             end
+
+            it 'returns prefilled 21-526EZ when disability_526_max_cfi_service_switch is enabled' do
+              Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_FOREGROUND)
+              Flipper.disable(:disability_compensation_remove_pciu)
+              Flipper.enable(:disability_526_max_cfi_service_switch)
```

Comment:
We're recommending a different way of turning flippers on and off in tests. [See example here](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample). Since the tests run in parallel in CI, enabling/disabling them like this can lead to flakey specs 😞.